Decisions/scs-xxxx-v1-ssd-flavors.md

```diff
@@ -0,0 +1,252 @@
+---
+title: SSD Flavors
+type: Decision Record
+status: Draft
+track: IaaS
+version: v0.1
```

Comment:
According to [SCS-0001#versioning](https://github.com/SovereignCloudStack/Docs/blob/main/Standards/scs-0001-v1-sovereign-cloud-standards.md#versioning), I'd advise to drop the version number within the frontmatter of this document. Minor changes (e.g. rewording or typo fixing) should be done without touching the version number. Major changes can be indicating by increasing the version number in the filename.

Reply:
Question is whether we want to allow for minor version updates, if we e.g. do editorial changes that clarifies the wording ... and do this in the version header without bumping the major version (which may carry new, in the worst case even incompatible changes). It's still good to reference the precise version number - which may be better to parse for humans that git hashes.

Reply:
@garloff *if* we use a version number, we should use semver imho.

Reply:
Fair comment. So major.minor.patch with patches probably being clarifications, while minor versions might introduce new backwards compatible requirements (i.e. someone fulfilling v1.8.x automatically fulfills also v1.y.z with y<=7) and major bumps even possibly having requirements that conflict with earlier versions?
If we do so, the filename should probably encode major.minor, no?